src/util/Websocket.ts

```diff
@@ -4,104 +4,117 @@ import WebSocket from "ws";
 import type { Client } from "..";
 
 export default class WS {
-    client: Client;
-    private ws?: WebSocket;
-    private client_id?: string;
-    private request_id: number = 1;
-    private channels: EventEmitter;
-    constructor(client: Client) {
```

Comment:
could you please fix all the weird indentation changes? it makes the diff very hard to read, and the rest of the codebase uses 4 spaces rather than 2